### ISSUE TITLE
[Bug] There is no option to select GPT-4 model.

### ISSUE THREAD

**Issue by jmartinezb3** [ChatGPTNextWeb/ChatGPT-Next-Web#3740]
When looking for the GPT-4 option in the model listing, there is no GPT-4 option, there are only GPT-3.5 options.

![imagen](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/94153214/5a1452fb-f1e4-46f3-9d7c-1f899d8c9341)

**Deployment**
https://app.nextchat.dev/ -v2.10.1

Mozilla Firefox.

**Comment by jmartinezb3** [comment]
@Yidadaa Sir, please can you help me with this issue?

**Comment by kkocdko** [comment]
Seems that they set `DISABLE_GPT4`? I'm not sure.

https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/blob/9eaf492d5b72d252c9dcaa55d5d99dab87bf19e0/README.md?plain=1#L209-L213

**Comment by MCheping8108** [comment]
I have added the `DISABLE_GPT4` environment variable, and GPT-4 has been disabled
![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/102772286/28ddd6c7-8fa2-4b98-9452-47ba6ddd895d)
![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/102772286/f4d4b384-5ac7-4ab9-814a-4d51000a9894)

**Comment by jmartinezb3** [comment]
> I have added the `DISABLE_GPT4` environment variable, and GPT-4 has been disabled ![image](https://private-user-images.githubusercontent.com/102772286/293898158-28ddd6c7-8fa2-4b98-9452-47ba6ddd895d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQyODQ1NzksIm5iZiI6MTcwNDI4NDI3OSwicGF0aCI6Ii8xMDI3NzIyODYvMjkzODk4MTU4LTI4ZGRkNmM3LThmYTItNGI5OC05NDUyLTQ3YmE2ZGRkODk1ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwM1QxMjE3NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOTIzN2U1ZTVjYTg1NjllZTljMDBhYjAxNDhhY2JiODZmZjY3YTM4YjhlNTBjYmJlZTY3OTZhOGM4ZWJjYTU0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.p12L8h03JVDdK8EcpcVWBxeKwdOM6bNXUmpjGmeVFM8) ![image](https://private-user-images.githubusercontent.com/102772286/293898186-f4d4b384-5ac7-4ab9-814a-4d51000a9894.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQyODQ1NzksIm5iZiI6MTcwNDI4NDI3OSwicGF0aCI6Ii8xMDI3NzIyODYvMjkzODk4MTg2LWY0ZDRiMzg0LTVhYzctNGFiOS04MTRhLTRkNTEwMDBhOTg5NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwM1QxMjE3NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kYWI4ZWYxYzQ4NWM2ZmNiMDI4MmU0MDNlMmNlMWVjYzYzYjc4YWYyNmM2YTk5MjdjN2QxMGQ5MjNiMzY4NGVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Lvf0RSuC1VCRnH7RGu6Az4yxINR4-Zo3UXwU9-6Jf5o)

I understand, do you mean that from now on it will not be possible to use gpt-4 in the webpage app.nextchat.dev?

**Comment by zero-cooper** [comment]
Any schedules when gpt4 can be reopened? I think this is a serious issue because most people are using gpt4 now.

**Comment by MCheping8108** [comment]
> Any schedules when gpt4 can be reopened? I think this is a serious issue because most people are using gpt4 now.gpt4何时可以重新开放？我认为这是一个严重的问题，因为现在大多数人都在使用gpt4。

go to here 👉 https://a.nextapi.fun/

**Comment by DasOhmoff** [comment]
Yeah, this really makes the app for many people unusable, including me :(

**Comment by DasOhmoff** [comment]
> > Any schedules when gpt4 can be reopened? I think this is a serious issue because most people are using gpt4 now.gpt4何时可以重新开放？我认为这是一个严重的问题，因为现在大多数人都在使用gpt4。
> 
> go to here 👉 https://a.nextapi.fun/

Hey thanks. Do you mind sharing from where is this link from and who hosts it?

**Comment by MCheping8108** [comment]
> > > Any schedules when gpt4 can be reopened? I think this is a serious issue because most people are using gpt4 now.gpt4何时可以重新开放？我认为这是一个严重的问题，因为现在大多数人都在使用gpt4。
> > 
> > 
> > go to here 👉 https://a.nextapi.fun/
> 
> Hey thanks. Do you mind sharing from where is this link from and who hosts it? (I wanna know if I can trust it)嘿，谢谢。你介意分享一下这个链接是从哪里来的，由谁托管的吗？（我想知道我是否可以信任它）


@Yidadaa
